spec/features/bulk_jobs_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+# Feature tests for the spreadsheet bulk uploads view.
+feature 'Bulk jobs view', js: true, focus: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.